alchemy/src/cloudflare/miniflare/remote-binding-proxy.ts

```diff
@@ -162,7 +180,12 @@ async function createWorkersPreviewSession(api: CloudflareApi) {
     "create workers preview session",
     api.get(`/accounts/${api.accountId}/workers/subdomain/edge-preview`),
```

Comment:
You might be wondering why we didn't include this [zone check](https://github.com/cloudflare/workers-sdk/blob/a5eb5134f73d3983655d325a4de71c6370c57faa/packages/wrangler/src/dev/create-worker-preview.ts#L162-L164):

```ts
ctx.zone
		? `/zones/${ctx.zone}/workers/edge-preview`
		: `/accounts/${accountId}/workers/subdomain/edge-preview`
```

My inclination is to support it in another PR iff it's necessary because it does add some extra complication.